superbench/benchmarks/micro_benchmarks/cuda_decode_performance/AppDecPerf.cpp

```diff
@@ -67,15 +67,19 @@ void ShowHelpAndExit(const char *szBadOption = NULL) {
         oss << "Error parsing \"" << szBadOption << "\"" << std::endl;
     }
     oss << "Options:" << std::endl
-        << "-i           Input single video file path" << std::endl
-        << "-o           Output file path" << std::endl
-        << "-gpu         Ordinal of GPU to use" << std::endl
-        << "-thread      Number of decoding thread" << std::endl
-        << "-total       Number of total video to test" << std::endl
-        << "-single      (No value) Use single context (default is multi-context, one context per thread)" << std::endl
-        << "-host        (No value) Copy frame to host memory (default is device memory)" << std::endl
-        << "-multi_input The file path which lists the path of multiple video in each line" << std::endl
-        << "-codec       The codec of video to test" << std::endl;
+        << "-i           Input file path. No default value. One of -i and -multi_input is required." << std::endl
+        << "-o           Output file path of raw data. No default value. Optional." << std::endl
+        << "-gpu         Ordinal of GPU to use. Default 0. Optional." << std::endl
+        << "-thread      Number of decoding thread. Default 5. Optional." << std::endl
+        << "-total       Number of total video to test. Default 100. Optional." << std::endl
+        << "-single      (No value) Use single cuda context for every thread. Default is multi-context, one context "
+           "per thread."
+        << std::endl
```

Comment:
Do we need to make "per thread." and << std::endl in single line?

Reply:
merge 'per thread' into last line but << std::endl is by clang-format